docs/en/operation/Upgrade.md

```diff
@@ -59,3 +59,26 @@ and then start all the masters with the local backup file path.
 Alluxio 2.x makes significant changes in the RPC layer,
 so pre-2.0.0 clients do not work with post-2.0.0 servers, and vice-versa.
 Upgrade all applications to use the alluxio-2.x client.
+
+## Additional Options
+
+### Alluxio worker ramdisk cache persistence
```

Comment:
This section is being added to reflect the additions added in https://github.com/Alluxio/alluxio/pull/13817